resources/levels/GameWorld.gd

```diff
@@ -0,0 +1,55 @@
+extends Node2D
+class_name GameWorld
+
+enum Biom {
+	BIOM_1
+}
+
+const mutator_effects = {
+	"time" : [0.95, 0.85, 0.72, 0.7, 0.65, 0.6],
+	"time_deviation" : [1, 1.1, 1.3, 0.8, 0.75, 0.6],
+	"size" : [1, 1, 1, 1, 1, 1],
+	"size_deviation" : [1, 1, 1, 1, 1, 1],
+	"max_weight" : [1, 1, 1, 1, 1, 1]
+}
+var current_biom = Biom.BIOM_1
```

Comment:
If we have this here we should use it over in TileMap.gd too as that also has a biome value. This can probably be used interchangeably.

Only thing is:  
Something like a "GameMaster"-class might store the Biome in the end which would mean two modifications so we could potentially leave it as is for now and then unify the systems via the game-master

Reply:
True. But yeah, let's leave it like that for now an see how things develop.